controlplane/pkg/monitor/monitor_server.go

```diff
@@ -33,6 +45,8 @@ type MonitorServer interface {
 	Update(entity Entity)
 	Delete(entity Entity)
 
+	UpdateStat(name string, statistics *interfaces.InterfacesState_Interface_Statistics)
```

Comment:
I believe this should be plural: `UpdateStats` or even better `UpdateStatistics`

Reply:
@nickolaev what do you think about bad abstractions here. I'd prefer to remove monitor server and use concrete implementation instead. Any thoughts on this?

Reply:
Or possibly UpdateMetrics ? (brainstorming :) )